module-system/sov-state/src/witness.rs

```diff
@@ -7,7 +7,7 @@ use serde::{Deserialize, Serialize};
 
 // TODO: Refactor witness trait so it only require Serialize / Deserialize
 //   https://github.com/Sovereign-Labs/sovereign-sdk/issues/263
-pub trait Witness: Default + Serialize {
+pub trait Witness: Default + Serialize + 'static {
```

Comment:
Question: Where did the new `'static` requirement come from?

Reply:
Resolved!